pox-locking/src/events.rs

```diff
@@ -195,11 +225,16 @@ fn create_event_info_data_code(
                         ;; derived from args[0]
                         total-locked: (+ {increase_by} (get locked (stx-account tx-sender))),
                         ;; pox addr increased
-                        pox-addr: (get pox-addr (unwrap-panic (map-get? stacking-state {{ stacker: tx-sender }})))
+                        pox-addr: (get pox-addr (unwrap-panic (map-get? stacking-state {{ stacker: tx-sender }}))),
+                        ;; Get end cycle ID
+                        end-cycle-id: (burn-height-to-reward-cycle unlock-height),
+                        ;; Get start cycle ID
+                        start-cycle-id: (+ (burn-height-to-reward-cycle unlock-height) prepare-offset),
                     }}
-                }}
+                }})
                 "#,
-                increase_by = &args[0]
+                increase_by = &args[0],
+                prepare_offset = prepare_offset.replace("%height%", "unlock-height"),
```

Comment:
With https://github.com/stacks-network/stacks-core/pull/4461 merging, we should also add signer-key to this `stacks-increase` event

Reply:
https://github.com/stacks-network/stacks-core/issues/4490